content/blog/2021-07-03-darktable-3.6/index.md

```diff
@@ -72,39 +72,39 @@ darktable 3.6 also introduces an OpenCL kernel for color calibration, which shou
 
 ## Censorization Module
 
-The censorization modules allows you to quickly blur and pixellate parts of an image, in an aesthetically-pleasing way, for anonymization purposes and to comply with social media regulations regarding body parts. 
+The censorization modules allows you to quickly blur and pixelate parts of an image, in an aesthetically pleasing way, for anonymization purposes and to comply with social media regulations regarding body parts. 
 
-It applies physically-realistic gaussian blur and gaussian noise in linear RGB. As such, it can also be used for various creative effects, including Orton effect (blooming). 
+It applies physically realistic Gaussian blur and Gaussian noise in linear RGB. As such, it can also be used for various creative effects, including Orton effect (blooming). 
 
 Note that the censorization module does not prevent forensic reconstruction of the censorized parts (using machine-learning and AI). For real forensic-level anonymization, you need to paint solid colors over the sensitive parts.
 
 ## Crop, Shapes, Retouch and Pipe Distortions
 
 A number of enhancements have been made so that the pipeline can better handle shapes and distorting modules.
 
-When shapes are drawn on the image they are actually drawn on the original raw image co-ordinates and then distorted through the pipe along with your image. This can mean, for example, that straight gradient lines appear curved, or circular masks are drawn as ellipses. In darktable 3.6 you will see these distortions in your drawn shapes _before_ you place them on your image and while you are editing any already-placed shapes. The retouch module has also been enhanced to better manage distortions in the source and target shapes. In addition, the gradient mask can now have its curvature set before it is placed on the image.
+When shapes are drawn on the image, they are actually drawn on the original raw image coordinates and then distorted through the pipe along with your image. This can mean, for example, that straight gradient lines appear curved, or circular masks are drawn as ellipses. In darktable 3.6 you will see these distortions in your drawn shapes _before_ you place them on your image and while you are editing any already-placed shapes. The retouch module has also been enhanced to better manage distortions in the source and target shapes. In addition, the gradient mask can now have its curvature set before it is placed on the image.
```

Comment:
For consistency, e.g. lines 123 & 153. Also, https://www.merriam-webster.com/dictionary/coordinate